runtime/src/bank_client.rs

```diff
@@ -331,7 +331,7 @@ impl BankClient {
         self.bank.set_sysvar_for_tests(sysvar);
     }
 
-    #[cfg(feature = "dev-context-only-utils")]
+    /// Only used for tests and benchmarks.
     pub fn advance_slot(
```

Comment:
This seems wrong? Why this change?

Reply:
This fn is now used in benches. Benches are not built in dev-context.  If it is decorated with dev-context-only, then benches build fails.

Reply:
credit to @pgarg66 
We can update the cargo.toml of bench to include dev-context-util dependency and keep the attribute.